int32 posAY
int32 posBX
float64 velAY
float64 velBX
float64 trqAY
float64 trqBX
float64 loadAY
float64 loadBX
float64 voltM
float64 voltC
int32 inp
int32 outp
int32 warn
int32 alarm

